zerver/tornado/event_queue.py

```diff
@@ -154,6 +156,7 @@ def to_dict(self) -> Dict[str, Any]:
             pronouns_field_type_supported=self.pronouns_field_type_supported,
             linkifier_url_template=self.linkifier_url_template,
             user_list_incomplete=self.user_list_incomplete,
+            archived_streams=self.archived_streams,
```

Comment:
I don't see this client capability doing anything in the events code. 

I think we'd [discussed](https://chat.zulip.org/#narrow/stream/378-api-design/topic/archived.20streams/near/1720672) that it should do this:

> and the events system pretend `archived: true` streams don't exist like the existing API does

But I'm not sure that's actually implementable easily -- the events code won't know if the stream is archived without a moderate amount of surgery. I'll raise that in the API design conversation.

Reply:
I have removed the code related to it for now. I will update the PR with necessary changes following whatever is concluded from the discussion.